bc_obps/reporting/tests/service/test_report_activity_save_service/data.py

```diff
@@ -29,9 +29,9 @@
                                     "test_emission_number": 12345,
                                     "test_emission_bool": True,
                                     "test_emission_str": "test",
-                                    "methodology": "Default EV",
-                                    "methodology_field_1": 123,
-                                    "method_field_2": "Some description",
+                                    "methodology": {
+                                        "methodology": "Default HHV/Default EF",
```

Comment:
We should still pass some random additional fields to the methodology here (or the real ones for that one)